DigitalLearningSolutions.Web.IntegrationTests/AuthenticationWebApplicationFactory.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace DigitalLearningSolutions.Web.IntegrationTests
+{
+    using System.Linq;
+    using DigitalLearningSolutions.Data.DataServices.UserDataService;
+    using DigitalLearningSolutions.Web.IntegrationTests.TestHelpers;
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Mvc.Testing;
+    using Microsoft.Extensions.DependencyInjection;
+
+    public class AuthenticationWebApplicationFactory<TStartup>
+        : WebApplicationFactory<TStartup> where TStartup : class
+    {
+        protected sealed override IWebHostBuilder CreateWebHostBuilder()
+        {
+            return base.CreateWebHostBuilder();
+        }
+
+        protected override void ConfigureWebHost(IWebHostBuilder builder)
+        {
+            builder
+                .UseEnvironment("SIT")

```

Comment:
Does this do anything special? Would it change behaviour if this string were changed?

Reply:
Yes, this will ensure the right appSettings file is used.

Reply:
Ah, always wondered what that was for. Thanks!